docs/start/v2.md

```diff
@@ -994,14 +994,14 @@ module.exports = {
 };
 ```
 
-[future-flags]: ./api-development-strategy
+[future-flags]: ./future-flags
 [remix-config]: ../file-conventions/remix-config
 [flat-routes]: https://github.com/remix-run/remix/discussions/4482
-[meta-v2]: ../route/meta-v2
+[meta-v2]: ../route/meta
```

Comment:
I didn't get this one in my changes so if you merge latest dev into here I think this will still need to get merged 👍